### PR TITLE
Fix bazel examples

### PR DESCRIPTION
**Pull request by Alexandr-Solovev**
This pull request contain changes for enable kmeans and dbscan in bazel examples. Also, minor fixes for covariance examples.

### CODE REVIEW

File: examples/oneapi/dpc/source/covariance/cov_dense_batch.cpp [hunk 1]
```diff
@@ -30,11 +30,11 @@ namespace dal = oneapi::dal;
 void run(sycl::queue &q) {
     const auto input_file_name = get_data_path("covcormoments_dense.csv");
 
-    const auto input = dal::read<dal::table>(dal::csv::data_source{ input_file_name });
+    const auto input = dal::read<dal::table>(q, dal::csv::data_source{ input_file_name });
```
Comment (samir-nasibli) [inline examples/oneapi/dpc/source/covariance/cov_dense_batch.cpp:33]: 👍

**Comment by samir-nasibli** [comment]
Could you please provide little description for the PR?

File: examples/oneapi/dpc/onedal_lnx.lst [hunk 2]
```diff
@@ -27,7 +27,6 @@ ONEDAL = basic_statistics_dense_batch      \
          df_cls_traverse_model             \
          df_reg_hist_batch                 \
          df_reg_traverse_model             \
-         dbscan_brute_force_batch          \
```
Comment (samir-nasibli) [inline examples/oneapi/dpc/onedal_lnx.lst:30]: 👍

File: examples/oneapi/dpc/BUILD [hunk 3]
```diff
@@ -29,13 +29,25 @@ dal_example_suite(
...
+)
+
 dal_algo_example_suite(
     algos = [
         "basic_statistics",
         "covariance",
         "decision_forest",
         "kmeans",
-        "kmeans_init",
+        "dbscan",
```
Comment (Alexsandruss) [inline examples/oneapi/dpc/BUILD:50]: It's not obvious why kmeans_init was moved and dbscan added here and then removed in list without description

**Review by Alexsandruss — CHANGES_REQUESTED** [review]

File: examples/oneapi/dpc/BUILD (hunk 3 above)
Comment (Alexandr-Solovev) [reply examples/oneapi/dpc/BUILD:50]: kmeans_init was moved to separate dal_example_suite due with conflicts between kmeans and kmeans_init. Dbscan was added due with adding dbscan examples to our bazel examples.

**Comment by Alexsandruss** [comment]
This PR will not be merged until CI is extended to check it.

**Review by KulikovNikita — APPROVED** [review]

**Review by Alexsandruss — APPROVED** [review]

**Review by samir-nasibli — APPROVED** [review]

**Pull request merged**